code/parachain/runtime/dali/src/lib.rs

```diff
@@ -811,28 +811,36 @@ impl currency_factory::Config for Runtime {
 	type Balance = Balance;
 }
 
+parameter_types! {
+	pub const AssetNameMaxChars: u32 = 32;
```

Comment:
not sure if we need to increase this

Reply:
After discussing the names of tokens more, I think raising this to 64 for both Dali and Picasso is fair.

Reply:
3060374d5b893267bd2a3ec00e0535a898f81c53